app/assets/stylesheets/hackney_property.scss

```diff
@@ -11,3 +11,19 @@
 .hackney-property-tenure-orange {
   background: hackney-colour("warning");
 }
+
+.hackney-cautionary-contact {
+  color: white !important;
+}
+
+.hackney-cautionary-contact-table {
+  background-color: hackney-colour("warning");
+  display: inline-block;
+  margin: 10px;
+  width: 50%;
```

Comment:
so is the overflow intended? If not, can you please make a wrapper div for both the tenure info and cautionary contact?

Something like

```html
<div style="width: 50%">
  <tenure_info style="width: 100%">
  <cautionary_contact style="width: 100%">
</div>
```

so that they will never get unaligned?